packages/web/src/store/application/ui/withdraw-usdc/sagas.ts

```diff
@@ -212,7 +44,10 @@ function* doWithdrawUSDCCoinflow({
   'amount' | 'currentBalance'
 >) {
   const { track, make } = yield* getContext('analytics')
+  const audiusSdk = yield* getContext('audiusSdk')
+  const sdk = yield* call(audiusSdk)
```

Comment:
I forgot to mention that there's a `getSDK()` helper you can yield to get a fully resolved sdk in one line, if that's your thing.